algorithms/sorting.md

```diff
@@ -23,8 +23,11 @@
 - [Bubble Sort With Hungarian Folk Dance](https://www.youtube.com/watch?v=lyZQPjUT5B4)
 - [Merge Sort With German Folk Dance](https://www.youtube.com/watch?v=XaqR3G_NVoo)
 - A Deck of Cards
-- [Sorting Algorithms Conceptually](https://code.likeagirl.io/sorting-algorithms-conceptually-2bfbb3968388)
 
+### Supplemental Resources
+***These materials are optional, that is, they are not required.***
```

Comment:
Please remove line 28, and move the supplemental resources section to be last, after check for understanding. After that, I’ll merge 😊

Reply:
Alright @alodahl I've done that, thanks